src/modules/UsersDialogsFlags.rb

```diff
@@ -0,0 +1,30 @@
+module Yast
+  # TODO: when not called from perl, it can be common ruby lib
+  class  UsersDialogsFlagsClass < Module
+    def assign_start_dialog(name)
+      @start_dialog = name
+    end
+
+    # define dialog which should be used as first one, can be modified with {start_dialog=}
+    def start_dialog
+      # summary is starting dialog for installation
+      @start_dialog ||= "summary"
+    end
+
+    def assign_use_next_time(value)
```

Comment:
The usual prefixes are `get` and `set`, I'm fine with leaving out `get` but I'd rather prefer `set` instead of `assign`, e.g. use `set_use_next_time` here.